### ISSUE TITLE
Missing "dig" Dependency

### ISSUE THREAD

**Issue by javaisbetterthanpython** [jawj/IKEv2-setup#77]
Got an error with "dig: command not found"
I suspect it just needs an "apt-get install dnsutils" added somewhere but not sure. This was on a freshly deployed VPS running Ubuntu 18.04 LTS

```
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Processing triggers for systemd (237-3ubuntu10.6) ...
Processing triggers for rsyslog (8.32.0-1ubuntu4) ...

--- Configuration: VPN settings ---

Network interface: eth0
External IP: 6x.xx.xx.210

** Note: hostname must resolve to this machine already, to enable Let's Encrypt certificate setup **
Hostname for VPN (default: 6x.xx.xx.210.server.io): server.mydomain.com
./setup.sh: line 62: dig: command not found
```

**Comment by javaisbetterthanpython** [comment]
Also missing the "uuid" dependency apparently, which required me to apt-get install uuid-runtime

**Comment by grand** [comment]
if you are using the "LTS minimal" version of Ubuntu, dig isn't included

**Comment by jawj** [comment]
Fixed in #78
